### PR TITLE
move kubectl generated man docs to docs/user-guide/kubectl

### PR DESCRIPTION
**Pull request by mikedanese**
@bgrant0607

### CODE REVIEW

**Comment by bgrant0607** [comment]
cc people working on mungers: @lavalamp @erictune @thockin

**Comment by bgrant0607** [comment]
cc @RichieEscarez @MikeJeffrey @jlowdermilk in case there are additional ramifications of the move

Otherwise, LGTM. Thanks. Would be nice to have a README.md in user-guide/kubectl, but this is a good step

File: docs/user-guide/connecting-to-applications-port-forward.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+﻿<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
```
Comment (mikedanese) [inline docs/user-guide/connecting-to-applications-port-forward.md]: appearantly this whitespace change broke mungedocs.. let me investigate. @thockin @lavalamp

**Comment by jlowdermilk** [comment]
GKE surfaces these docs as well, but changing root dir is minor. LGTM

**Pull request merged**